testsuite/integration-arquillian/pom.xml

```diff
@@ -309,6 +309,20 @@
                             </rules>
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>enforce-maven</id>
```

Comment:
nit: shouldn't it be different from other execution ids? what about `enforce-maven-deploy`? But it's rather optional, feel free to ignore.

Reply:
+1

Reply:
done